index.html

```diff
@@ -0,0 +1,242 @@
+                        <img src="assets/images/event-vect.png" alt="">
+                    </div>
+
+                    <div class="">
+                        <h2>
+                            fashion show
+                        </h2>
+                    </div>
+                </div>
+
+                <div class="leading">
+                    <p>
+                        don't miss our upcoming fashion show and discover new talents
+                        from everywhere.
+                    </p>
+                </div>
+
+            </div>
+
+
+            <div class="event-card">
+                <div class="trailing">
+                    <div class="">
+                        <img src="assets/images/event-vect2.png" alt="">
+                    </div>
+
+                    <div class="">
+                        <h2>Exhibition</h2>
+                    </div>
+                </div>
+
+                <div class="leading">
+                    <p>
+                        Come and participate at our products exhibition, and
+                    </p>
+                </div>
+
+            </div>
+
+
+            <div class="event-card">
+                <div class="trailing">
+                    <div class="">
+                        <img src="assets/images/event-vect3.png" alt="">
+                    </div>
+
+                    <div class="">
+                        <h2>Forum</h2>
+                    </div>
+                </div>
+
+                <div class="leading">
+                    <p>
+                        Have the time to share your thoughts and opinions with experts for each topic.
+                    </p>
+                </div>
+
+            </div>
+
+            <div class="event-card">
+                <div class="trailing">
+                    <div class="">
+                        <img src="assets/images/event-vect4.png" alt="">
+                    </div>
+
+                    <div class="">
+                        <h2>Workshop</h2>
+                    </div>
+                </div>
+
+                <div class="leading">
+                    <p>
+                        Try creating your own work using open source license rather than just watching at it.
+                    </p>
+                </div>
+
+            </div>
+        </div>
+
+        <div class="event-btn-container">
+            <button>Join CC Global Summit 2015</button>
+        </div>
+
+        <div class="lg-event-link-container">
+            <a href="">SEE THE WHOLE PROGRAM</a>
+        </div>
+
+    </section>
+
+
+    <section class="mobile-team">
+        <div class="team-mobile-list-wrapper">
+            <h2 class="expert-title">Our experts</h2>
+            <ul class="mobile-team-list">
+
+            </ul>
+        </div>
+
+        <button class="team-btn">
+            <p>MORE</p>
+            <i class='bx bxs-chevron-down'></i>
+        </button>
+    </section>
+
+    <section class="lg-screen-team">
```

Comment:
[Optional] Kindly add some bottom padding to your featured speakers section to improve the look of the design
<img width="498" alt="image" src="https://github.com/DjibrilM/capstone-project/assets/37116322/e06f7c04-e5f7-438c-8209-861b93290dfb">